examples/authorization/create_authorization_with_account.rb

```diff
@@ -0,0 +1,56 @@
+require_relative "../boot"
+
+# Create Authorization with Account
+#
+#   You need to give:
+#     - notification_url
+#     - redirect_url
+#     - permissions defaults to all permissions
+#     - applications credentials (APP_ID, APP_KEY)
+#     - account params
+#
+#   You can pass this parameters to PagSeguro::AuthorizationRequest#new
+#
+# PS: For more details look the class PagSeguro::AuthorizationRequest
```

Comment:
Some typos
`applications credentials` > `application credentials`
`You can pass this...` > `You can pass these...`
`...look the class...` > `...take a look at the class...`